openconnect-gp-okta

```diff
@@ -30,6 +31,17 @@ except ImportError:
     HAS_FIDO2 = False
     UserInteraction = object
 
+HAS_PKCS12 = True
```

Comment:
nit: it seems slightly neater to me to do

```
try:
    from ... import ...
    HAS_PKCS12 = True
except ImportError:
    HAS_PKCS12 = False

Reply:
Done/